roles/inspec/defaults/main.yml

```diff
@@ -9,9 +9,9 @@ inspec:
   dependency_path: /etc/inspec/
   profiles:
     openstack_security:
-      name: inspec-openstack-security
+      name: openstack-baseline
```

Comment:
We need to rename the profile because of the name. We can do profiles_to_purge if we want to do clean up. Two directories will exist for the same profile. But what name is included in `openstack-security.name` will be ran but not both.